Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -453,6 +489,18 @@ void EnhancementsWidget::AddDescriptions()
       "scaling filter selected by the game.<br><br>Any option except 'Default' will alter the look "
       "of the game's textures and might cause issues in a small number of "
       "games.<br><br><dolphin_emphasis>If unsure, select 'Default'.</dolphin_emphasis>");
+  static const char TR_OUTPUT_RESAMPLING_DESCRIPTION[] = QT_TR_NOOP(
+      "Affects how the game output image is upscaled or downscaled to the window resolution.<br>"
+      "\"Default\" will rely on the GPU internal bilinear sampler which isn't gamma corrected."
+      "<br>\"Bilinear\" (gamma corrected) is a good compromise between quality and performance."
+      "<br>\"Bicubic\" is smoother than \"Bilinear\"."
+      "<br>\"Hermite\" might offer the best quality when upscaling,"
+      " at a slightly bigger perform cost.<br>\"Catmull-Rom\" is best for downscaling."
+      "<br>\"Nearest Neighbor\" doesn't do any resampling, select if you like a pixelated look."
```

Comment:
So um, why is nearest neighbor even here? Like, I know it's cheap and easy, but it has a lot of downsides and artifacts.... and we have sharp bilinear. We should be pushing anyone who likes the pixelated look toward sharp bilinear. I don't think we should even be giving Nearest Neighbor as an option to select in this menu.

Reply:
I don't see any reason to not expose it. We can put it last in the order?

Reply:
Since you are adding something new, the opposite approach should be used, imo. The addition needs to be justified. In this case, why would we expose a setting we don't want users to use? That we know is worse than another option we have? That sounds like a perfect INI only option to me!

Reply:
@AdmiralCurtiss do you agree that not having Nearest Neighbor is good and that Sharp Bilinear is better for all cases? If so, I'm on with not exposing it.